src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,144 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node first;
+    private Node last;
+    private int size;
+
     @Override
     public boolean add(T value) {
-        return false;
+        Node<T> t = last;
```

Comment:
let's rename variable `t` and other the same variables

Reply:
done
now it`s 'currentLast'